modules/util/portal-tools-rest-builder-test-test/src/testIntegration/java/com/liferay/portal/tools/rest/builder/test/resource/v1_0/test/BaseTestEntityResourceTestCase.java

```diff
@@ -582,6 +582,11 @@ protected TestEntity testPostTestEntity_addTestEntity(TestEntity testEntity)
 			"This method needs to be implemented");
 	}
 
+	@Test
+	public void testPostTestEntityMultiformBulk() throws Exception {
+		Assert.assertTrue(false);
```

Comment:
I think this test is going to start failing unless you ignore it or implement it